### PR TITLE
Upload storageclass-accessor

### PR DESCRIPTION
**Pull request by f10atin9**
Signed-off-by: f10atin9 <f10atin9@kubesphere.io>

### CODE REVIEW

File: src/main/storageclass-accessor/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+helm delete storageclass-accessor --namespace storageclass-accessor
+```
+
+## Configuration
+
+The following table lists the configurable parameters of the chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`image.repository` | Image repository of accessor deployment | `f10atin9/storageclass-accessor`
```
Comment (stoneshi-yunify) [inline src/main/storageclass-accessor/README.md:33]: use kubesphere image repo

**Review by stoneshi-yunify — APPROVED** [review]
/lgtm

**Comment by zheng1** [comment]
CI meets error:
```
Error: 1 chart(s) linted, 1 chart(s) failed
==> Linting src/main/storageclass-accessor
[INFO] Chart.yaml: icon is recommended
Error:  Chart.yaml: chart type is not valid in apiVersion 'v1'. It is valid in apiVersion 'v2'
```

**Pull request merged**